shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -370,12 +370,38 @@ impl CassandraSinkCluster {
             super::connection::receive(self.read_timeout, &self.failed_requests, responses_future)
                 .await?;
 
-        let _prepare_responses = super::connection::receive(
-            self.read_timeout,
-            &self.failed_requests,
-            responses_future_prepare,
-        )
-        .await?;
+        {
+            let mut prepare_responses = super::connection::receive(
+                self.read_timeout,
+                &self.failed_requests,
+                responses_future_prepare,
+            )
+            .await?;
+
+            if !prepare_responses.windows(2).all(|w| w[0] == w[1]) {
+                let err_str = prepare_responses
```

Comment:
put the err_str in the branch its used in

Reply:
Actually maybe we should use err_str in the tracing::error as well

Reply:
Oops, intended to use it in both.